src/components/Sections/SectionMarkdown.js

```diff
@@ -23,176 +23,178 @@ import ins from 'markdown-it-ins';
 
 const IGNORE_KEYS = [PAGE_BREAK_KEY];
 
-function createBtn(props, children) {
-  let message = children;
-  try {
-    const obj = JSON.parse(children[0]);
-    message = obj.message || message;
-  } catch (e) {
-    // do nothing
-  }
-  return (<span {...props}>{message}</span>);
-}
+export default class SectionMarkdown extends Component {
+  static propTypes = {
+    text: PropTypes.string,
+    style: PropTypes.object,
+    setRef: PropTypes.any,
+    tableClasses: PropTypes.oneOfType([
+      PropTypes.object,
+      PropTypes.string
+    ])
+  };
 
-
-function handleIterate(tableClasses, Tag, props, children) {
-  let res = '';
-  switch (Tag) {
-    case 'p':
-      res = (<div className="preplacer" {...props}>{children}</div>);
-      break;
-    case 'hr':
-      res = (<hr {...props} />);
-      break;
-    case 'mdbtn': {
-      res = createBtn(props, children);
-      break;
+  static createBtn(props, children) {
+    let message = children;
+    try {
+      const obj = JSON.parse(children[0]);
+      message = obj.message || message;
+    } catch (e) {
+      // do nothing
     }
-    case 'img': {
-      const srcArr = props.src.split('=size=');
-      props.src = srcArr[0];
-      if (srcArr.length > 1) {
-        const sizeArr = srcArr[1].split('x');
-        props.height = sizeArr[0];
-        if (sizeArr.length > 1) {
-          props.width = sizeArr[1];
-        }
+    return (<span {...props}>{message}</span>);
+  }
+
+  static handleIterate(tableClasses, Tag, props, children) {
+    let res = '';
+    switch (Tag) {
+      case 'p':
+        res = (<div className="preplacer" {...props}>{children}</div>);
+        break;
+      case 'hr':
+        res = (<hr {...props} />);
+        break;
+      case 'mdbtn': {
+        res = SectionMarkdown.createBtn(props, children);
+        break;
       }
+      case 'img': {
+        const srcArr = props.src.split('=size=');
```

Comment:
what is going on here? ('=size='?)

Reply:
existing option to resize image in markdown similar to https://gist.github.com/uupaa/f77d2bcf4dc7a294d109